autoPyTorch/api/base_task.py

```diff
@@ -583,16 +724,47 @@ def search(
             raise ValueError("Cannot interpret task type from the dataset")
 
         self._create_dask_client()
+
+        # ============> Run dummy predictions
+        num_run = 1
+        dummy_task_name = 'runDummy'
+        self._stopwatch.start_task(dummy_task_name)
+        self._do_dummy_prediction(num_run)
+        self._stopwatch.stop_task(dummy_task_name)
+
+        # ============> Run traditional ml
+
+        traditional_task_name = 'runTraditional'
+        self._stopwatch.start_task(traditional_task_name)
+        elapsed_time = self._stopwatch.wall_elapsed(self.dataset_name)
+        time_for_traditional = int(traditional_per_total_budget * max(0, (self._time_for_task - elapsed_time)))
+        if time_for_traditional <= 0:
+            if traditional_per_total_budget > 0:
+                raise ValueError("Not enough time allocated to run traditional algorithms")
+        elif traditional_per_total_budget != 0:
+            num_run = self._do_traditional_prediction(num_run=num_run, time_for_traditional=time_for_traditional)
```

Comment:
Shouldn't this be:
```
 num_run = self._do_traditional_prediction(num_run=num_run+1, time_for_traditional=time_for_traditional)
```

The num_run+1  to account for dummy predictions?

Reply:
yeah sorry.